src/tests/application/encryptor_tests.cpp

```diff
@@ -1,8 +1,8 @@
-#include <gtest/gtest.h>
-#include "../../../application/DDD/PasswordEncryption.h"
-#include "../../../application/Cipher.h"
-#include "../../../plugins/encryption/CBC_Cipher.h"
-#include "../../../plugins/encryption/BLOCK.h"
+#include "../../cmake-build-debug/_deps/googletest-src/googletest/include/gtest/gtest.h"
```

Comment:
I'm guessing this fixes the CLion build?

Reply:
It does not. It was somehow automatically expanded into that from <gtest/gtest.h> by CLion. Thanks for pointing that out, looks pretty ugly.